instance/models/mixins/openedx_config.py

```diff
@@ -65,6 +67,12 @@ def _get_configuration_variables(self):
             # We cannot do this on OC IM for security reasons (we don't want different *instances* to share cookies).
             "EDXAPP_SESSION_COOKIE_DOMAIN": '',
 
+            # Run a command to delete expired sessions once a day. The time is random and different in each server
+            # to avoid the case when all servers connect to the database at exactly the same time
+            "EDXAPP_CLEARSESSIONS_CRON_ENABLED": True,
+            "EDXAPP_CLEARSESSIONS_CRON_HOURS": randint(0, 23),
```

Comment:
I had to double check that this method is only called once for each app server, and the resulting configuration is stored in `configuration_settings`, so it's fine that it isn't consistent across multipe calls.  Maybe add a comment expliaining this here?

Reply:
Ok, I added the comment.